packages/keystatic/src/app/shell/topbar.tsx

```diff
@@ -109,6 +111,7 @@ function CloudHeader({ config }: { config: CloudConfig }) {
       <BranchPicker />
       <GitMenu />
       <Box flex="1" />
+      <ImageLibraryButton />
```

Comment:
We should also have this in the `LocalHeader`

Reply:
Good catch, thanks!